### PR TITLE
Fix Sourcegraph casing

### PR DESCRIPTION
**Pull request by ryanslade**
I guess we can't expect others to get it right if we don't :)

### CODE REVIEW

**Comment by ryanslade** [comment]
@unknwon Special one for you, I know how you love consistent casing :)

File: internal/cmd/init-sg/initSourceGraph.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/sourcegraph/sourcegraph/internal/gqltestutil"
 )
```
Comment (unknwon) [inline internal/cmd/init-sg/initSourceGraph.go:8]: File name also needs to change 😁

**Review by unknwon — APPROVED** [review]

File: internal/cmd/init-sg/initSourceGraph.go (hunk 1 above)
Comment (ryanslade) [reply internal/cmd/init-sg/initSourceGraph.go:8]: Yes it does!

Comment (ryanslade) [reply internal/cmd/init-sg/initSourceGraph.go:8]: I decided to just move everything into main.go, no need for all the small files.

**Pull request merged**